src/site/content/en/reliable/instant-navigation-experiences/index.md

```diff
@@ -0,0 +1,166 @@
+{% endAside %}
+
+### 1. Precache static pages and page subresources
+
+[Precaching](https://web.dev/precache-with-workbox/) is the ability of the service worker to save files to the cache while it's installing. 
+
+{% Aside %}
+Precaching sounds similar to prefetching, but it's a different technique. In the first one, the service worker fetches and stores resources (typically static files) while it's installing and keeps them in the cache until a new version of the file is available. In the second, resources are requested ahead of time to have it in the cache for brief periods of time in order to speed up subsequent navigations. 
+{% endAside %}
+
+In the following cases precaching is used to achieve similar goal than prefetching: making navigations faster.
+
+#### Precaching static pages
+
+For pages that are generated at build time (e.g. `about.html`, `contact.html`), or in completely static sites, one can just add the site's documents to the precache list, so they are already available in the cache every time the user accesses them:
+
+```javascript
+workbox.precaching.precacheAndRoute([
+  {url: '/about.html', revision: 'abcd1234'},
+  // ... other entries ...
+]);
+```
+
+#### Precaching page subresources 
+
+Precaching static assets that the different sections of the site might use (e.g. JavaScript, CSS, etc.), is a general best practice and can give an extra boost in prefetching scenarios.
+
+To speed up navigations in an e-commerce site, you can use  `<link rel="prefetch">` tags in listing pages to prefetch product detail pages for the first few products of a listing page. If you have already precached the product page subresources, this can make the navigation even faster.
+
+To implement this:
+
+- Add a `<link rel="prefetch">` tag to the page:
+
+```html
+ <link rel="prefetch" href="/phones/smartphone-5x.html" as="document">
+```
+
+- Add the page subresources to the precache list in the service worker:
+
+```javascript
+workbox.precaching.precacheAndRoute([
+  '/styles/product-page.ac29.css',
+  // ... other entries ...
+]);
+```
+
+### 2. Extend the lifetime of prefetch resources
+
+As mentioned earlier, `<link rel="prefetch">` fetches and keeps resources in the HTTP cache for a limited amount of time, after which point the `Cache-Control` rules for a resource apply. As of Chrome 56, this value is 5 minutes.
```

Comment:
```suggestion
As mentioned earlier, `<link rel="prefetch">` fetches and keeps resources in the HTTP cache for a limited amount of time, after which point the `Cache-Control` rules for a resource apply. As of Chrome 85, this value is 5 minutes.
```

Reply:
I'm just thinking about this now: do we have any knowledge about how the other browsers behave? That would be nice to include. This isn't a blocker but if you do know this information, it would be nice to include.

Reply:
Hi Kayce, this is an important point. As this is not part of the spec, each browser can implement the prefetching window differently, and there's no public information about it. At the time when we wrote https://web.dev/link-prefetch/ (the canonical guide), we were able to get this info from Chrome folks, but unfortunately couldn't find details about the rest.